container.go

```diff
@@ -32,12 +32,6 @@ type ContainerConfig struct {
 	// RootFs is the container workload image on the host.
 	RootFs string
 
-	// Interactive specifies if the container runs in the foreground.
-	Interactive bool
-
-	// Console is a console path provided by the caller.
-	Console string
-
```

Comment:
Could you please explain why we can't have a tty per container ?
`Interactive` might not make sense, but tty for stderr/out and signaling is relevant on a per containers basis.